packages/@o3r/testing/schematics/ng-add/templates/testing/setup-jest.ts.template

```diff
@@ -0,0 +1 @@
+import 'jest-preset-angular/setup-jest';
```

Comment:
Maybe we should make sure (in the `ng-add`) that the `jest-preset-angular` is installed

Reply:
have a look at `packages/@o3r/testing/schematics/ng-add/index.ts`:  `ngAddPeerDependencyPackages(['jest', 'jest-preset-angular'], testPackageJsonPath, NodeDependencyType.Dev, options),
`